presto-main/src/main/java/com/facebook/presto/server/ResourceUtil.java

```diff
@@ -280,4 +278,24 @@ private static String trimEmptyToNull(String value)
     {
         return emptyToNull(nullToEmpty(value).trim());
     }
+
+    public static String urlEncode(String value)
+    {
+        try {
+            return URLEncoder.encode(value, "UTF-8");
+        }
+        catch (UnsupportedEncodingException e) {
+            throw new AssertionError(e);
+        }
+    }
+
+    private static String urlDecode(String value)
```

Comment:
We should also make this one public, to be consistent (even though it's not used outside)